MCprep_addon/spawner/mcmodel.py

```diff
@@ -484,7 +535,11 @@ def execute(self, context):
 			return {'CANCELLED'}
 
 		try:
-			obj = add_model(os.path.normpath(self.filepath), name)
+			r, obj = add_model(os.path.normpath(self.filepath), name)
+			if r:
+				self.report(
+					{"ERROR"}, "The JSON defined is not valid for a Minecraft Java Edition model")
```

Comment:
```suggestion
					{"ERROR"}, "The JSON model is not avalid JSON model as per Minecraft Java Edition standards")
```

Reply:
How about "The JSON model is not valid for Minecraft Java Edition"
Or something like "The JSON model does not contain any geometry elements" since it only return that because it doesn't have any geometry information.